### PR TITLE
[Term Entry] Pillow library method: .Transpose()

### LINKED ISSUE #4625: [Term Entry] Python pillow .transpose()
### Reason/inspiration (optional)

We would like a new term entry on the '.transpose()' method for the `Pillow` library in Python. The entry should go in a new file under `docs/content/pillow/concepts/image/terms/transpose/transpose.md`

The entry should include the following:

- A description of the function
- A ## Syntax section that includes the syntax of the function and its details
- An ## Example section that includes a ```python ``` codeblock and ```shell ``` that shows the example output.

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by DevIggor**
### Description

-Added the .Transpose() term for Pillow
-Closes Issue #4625 
### Issue Solved
Closes #4625 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by DevIggor** [comment]
@cigar-galaxy82 Hey 👋 I have already signed the CLA and my email is also linked in my account and my bash command line but I still get the message from the CLA Assistant. Is there anything wrong my contribution?

**Comment by cigar-galaxy82** [comment]
Hi @DevIggor  don't worry about the CLA sometimes our CLAassistant behaves unpredictably

**Comment by mamtawardhani** [comment]
Hey @DevIggor, it seems like when you raised this PR, you were signed out of your GitHub account, I suppose. 
You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

Could you please fill this out?

**Comment by DevIggor** [comment]
Hey @mamtawardhani 👋 Just rechecked my email and my github account, it's all setted up and my Email is also verified and linked to my account, I sign in and out just to make sure there's nothing wrong on my side

File: content/pillow/concepts/image/terms/transpose/transpose.md
Comment (cigar-galaxy82) [inline content/pillow/concepts/image/terms/transpose/transpose.md:50]: I think it would be good if you could add a before and after image to show what kind of output one can expect after running this code

File: content/pillow/concepts/image/terms/transpose/transpose.md
Comment (cigar-galaxy82) [inline content/pillow/concepts/image/terms/transpose/transpose.md:14]: Also, add these two in CatalogContent
  - 'paths/data-science'
  - 'paths/data-science-foundations'

File: content/pillow/concepts/image/terms/transpose/transpose.md
Comment (cigar-galaxy82) [inline content/pillow/concepts/image/terms/transpose/transpose.md:17]:
```suggestion
The **.transpose()** method is used to flip or rotate an image in 90-degree steps.
```

File: content/pillow/concepts/image/terms/transpose/transpose.md
Comment (cigar-galaxy82) [inline content/pillow/concepts/image/terms/transpose/transpose.md:26]:
```suggestion
# Where possible values of the method are
```

File: content/pillow/concepts/image/terms/transpose/transpose.md
Comment (cigar-galaxy82) [inline content/pillow/concepts/image/terms/transpose/transpose.md:10]: Add Pillow in the tags

**Review by cigar-galaxy82 — CHANGES_REQUESTED** [review]
Hi, I have suggested a few changes please modify them

**Comment by cigar-galaxy82** [comment]
Hey @DevIggor 

Thank you for contributing to Codecademy Docs! The entry is well written!

I've suggested a few changes please modify them

Thank you!

**Comment by DevIggor** [comment]
> Hey @DevIggor
> 
> Thank you for contributing to Codecademy Docs! The entry is well written!
> 
> I've suggested a few changes please modify them
> 
> Thank you!

changes made, thank you for the feedback!. 

Btw CLAassistant's still not approvingm me 😔

**Comment by letsbuildanything** [comment]
Hey @DevIggor,
It's important to resolve the CLAssistant warning before merging this PR. Please carefully follow the instructions provided by CLAassistant.
Thank you!

**Comment by mamtawardhani** [comment]
@letsbuildanything I guess @DevIggor did the required steps, however it still doesn't work for him. Should he raise another PR and close this one, for quick closing of this issue? What do you suggest?

**Comment by letsbuildanything** [comment]
Hey @DevIggor, I've noticed that you're using two different accounts::
1: Author: DevIggor <DevIggor.010110@gmail.com>
2: Author: Igor Junio <dev.iggor010110@gmail.com>

And, for this commit [Create term entry for transpose method in pillow](https://github.com/Codecademy/docs/pull/4703/commits/dcb2c29f1acb6062a600c73aecd3450b66a7131f), you've used the account linked with first email address(Author: DevIggor <DevIggor.010110@gmail.com>). Please ensure that you've signed the CLA using both the account.

**Comment by DevIggor** [comment]
> Hey @DevIggor, I've noticed that you're using two different accounts:: 1: Author: DevIggor [DevIggor.010110@gmail.com](mailto:DevIggor.010110@gmail.com) 2: Author: Igor Junio [dev.iggor010110@gmail.com](mailto:dev.iggor010110@gmail.com)
> 
> And, for this commit [Create term entry for transpose method in pillow](https://github.com/Codecademy/docs/pull/4703/commits/dcb2c29f1acb6062a600c73aecd3450b66a7131f), you've used the account linked with first email address(Author: DevIggor [DevIggor.010110@gmail.com](mailto:DevIggor.010110@gmail.com)). Please ensure that you've signed the CLA using both the account.

Will do that now, thank you :)

**Comment by letsbuildanything** [comment]
@mamtawardhani Let's wait for @DevIggor's response first to confirm if the issue persists. We can decide the next steps based on his feedback.

**Comment by letsbuildanything** [comment]
@DevIggor Thank you for your cooperation!
Could you please check if your Git account is configured for this PR in your local terminal by using the following commands:
- `git config user.email`
- `git config user.name`
If these are not configured, please set them using the commands below:
`git config --global user.email <your_correct_email@example.com>`
`git config --global user.name <your_correct_username>`

**Comment by letsbuildanything** [comment]
@DevIggor Now, If your Git account is configured with the same email, please try committing some minor changes to your file and pushing it to see if this resolves the issue.

If the problem persists, I recommend closing this PR and submitting a new one with all the necessary changes. Be sure to request a review from @cigar-galaxy82 on your new PR.

Thank you!

**Comment by DevIggor** [comment]
> @DevIggor Now, If your Git account is configured with the same email, please try committing some minor changes to your file and pushing it to see if this resolves the issue.
> 
> If the problem persists, I recommend closing this PR and submitting a new one with all the necessary changes. Be sure to request a review from @cigar-galaxy82 on your new PR.
> 
> Thank you!

Minor changes made. Awaiting for review, In case this doesn't work I'll close this PR and will be submitting a new one.

**Comment by letsbuildanything** [comment]
@DevIggor 
Please close this PR and resubmit a new one. Apologies for any inconvenience this may cause.

**Pull request closed without merging**
